scripts/migrate_quotes.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import csv
+import os
+import pathlib
+
+import requests
+
+NB2_PORT = os.environ.get("NB2_PORT", "8000")
+
+
+def get_url(url):
+    return f"{'http' if parsed_args.use_http else 'https'}://127.0.0.1:{NB2_PORT}{url}"
+
+
+def post(url, data):
+    response = requests.post(get_url(url), data)
+    if response.status_code > 400 and response.status_code != 409:
```

Comment:
409 is the error we raise when a person or quote already exists right? (Just confirming)